README.md

```diff
@@ -111,9 +111,9 @@ Contact the project developers via the project's "dev" list.
 Search for bugs:
 ----------------
 
-SWT used to track ongoing development and issues in Bugzilla .
+Github is used to track current issues
 
-- <https://bugs.eclipse.org/bugs/buglist.cgi?product=Platform&component=SWT>
+- <https://github.com/issues?user=eclipse-platform>
```

Comment:
The URL for SWT should be `https://github.com/eclipse-platform/eclipse.platform.swt/issues`

But is there any point in this? This README file is on the front page of the GH with the "Issues" menu right at the top of the page.

Reply:
So I think it doesn't make sense. Then you can remove this section entirely, as one of the reviewers advised me to do